appveyor.yml

```diff
@@ -1,4 +1,5 @@
 version: '{build}'
+image: Visual Studio 2017
 build_script:
 - ps: ./Build.ps1 -majorMinor "3.0" -patch "$env:APPVEYOR_BUILD_VERSION" -customLogger "C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
Given the binary breaking change, should we bump this to 4.0?

Reply:
yep I'll do that :)